### DISCUSSION TITLE
WARNING: Host is already in use by another container

### DISCUSSION THREAD

**Discussion by MyBlood** [mediacms-io/mediacms#453] [Q&A] [answered]
Greetings all!

How can I solve this problem?
![image](https://user-images.githubusercontent.com/20350848/167680647-6a40f4af-4101-40e3-94fd-4a7a29cf278b.png)

**Comment by mgogoulos** [comment, marked as answer]
Something is running on 443 on your system, you might want to stop it and restart mediacms

**Reply by MyBlood** [reply]
> Something is running on 443 on your system, you might want to stop it and restart mediacms

Yes, this port is occupied by the server control panel. Can I change this port for the mediacms container?
